### PR TITLE
feat: set displayName and __ui__ for color-picker

### LINKED ISSUE #2861: `displayName` is not set for the component of "@yamada-ui/color-picker"
script: component-display-name
package: @yamada-ui/color-picker

### Description
The component "@yamada-ui/color-picker" uses `forwardRef`, but `displayName` is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

### Additional Information
Set `displayName` and `__ui__` after each component definition.

For example, like this.

```tsx
const Component = forwardRef((props, ref) => {
  ...
})

Component.displayName = 'Component'
Component.__ui__ = 'Component'
```

Replace `'component'` with the name of the component. For example, for a `Select` component, it is `'Select'`.

`__ui__` is an identifier required for internal use by Yamada UI. Be sure to set it together with `displayName`.

### PR DESCRIPTION
**Pull request by yuto-trd**
Closes #2861

## Description

Set `displayName` and `__ui__` for each component in the color picker.

### CODE REVIEW

**Comment by cidkumagai** [comment]
@yuto-trd 
Hi! Thank you for PR.
I think this issue can be resolved by only making changes to `forwardRef`.

File: packages/components/color-picker/src/color-selector-eye-dropper.tsx [hunk 1]
```diff
@@ -54,3 +57,5 @@ export const EyeDropperIcon: FC<IconProps> = ({ ...rest }) => {
     </Icon>
   )
 }
+
+EyeDropperIcon.displayName = "EyeDropperIcon"
```
[the remark this answers is not in the document]
Comment (yuto-trd) [reply packages/components/color-picker/src/color-selector-eye-dropper.tsx:61]: I see. I'll fix it.

[the remark this answers is not in the document]
Comment (yuto-trd) [reply packages/components/color-picker/src/color-selector-eye-dropper.tsx]: @cidkumagai 
I fixed it. Please review again.

**Review by cidkumagai — APPROVED** [review]
LGTM!!

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**